amgut/templates/new_participant.html

```diff
@@ -131,7 +131,7 @@ <h3>{% raw tl['SEL_AGE_RANGE'] %}</h3>
 
 <div id="0-6-div" style='text-align: left;padding: 10px'>
 <div class='scrolltext'>
-{% raw tl['CONSENT_YOUR_CHILD']%}
+{% raw tl['CONSENT_YOUR_CHILD_UPDATE']%}
```

Comment:
Is the `CONSENT_YOUR_CHILD` key no longer relevant?

Reply:
I will go back and check what I did here. I put the CONSENT_YOUR_CHILD_UPDATE key when I was figuring out how html worked and wanted to make sure I wasn't breaking things. I will change it so that only the CONSENT_YOUR_CHILD key is there, I didn't add _UPDATE to the other keys, I don't think it makes sense to only have changed that one.

Reply:
okay thanks!